modules/cassandra/executable.go

```diff
@@ -0,0 +1,18 @@
+package cassandra
+
+import (
+	"strings"
+)
+
+type initScript struct {
+	File string
+}
+
+func (i initScript) AsCommand() []string {
+	if strings.HasSuffix(i.File, ".cql") {
+		return []string{"cqlsh", "-f", i.File}
+	} else if strings.HasSuffix(i.File, ".sh") {
+		return []string{"/bin/sh", i.File}
+	}
+	return nil
+}
```

Comment:
@mdelapenya hey, can you review this again? I add this to the separated file as "executable" but I need help with the naming convention because there is no example except rabbitmq/types_test.go. I think we should decide on good naming for both file names and structs for this executable usage for the sake of example to other modules

Reply:
I think `initScript`, not exported, is fine. I thought about the following names:
- `cassandraCommand`: no because it includes the package namespace, and no again because calling the .AsCommand() method will come with duplication.
- `script`: no because it's very simple
- `executable`: no because it's very abstract
- `initScript`:  in the context of Cassandra, I think this is the one that makes more sense.